packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -98,11 +112,18 @@ class CupertinoSegmentedControl<T extends Object> extends StatefulWidget {
     this.selectedColor,
     this.borderColor,
     this.pressedColor,
+    this.disabledColor,
+    this.disabledTextColor,
     this.padding,
+    this.segmentStates = const <int, bool>{},
```

Comment:
Would it be better if we use `<T, bool>`, similar to the `children` property?

Reply:
Thank you for the suggestion. You are right, it looks better with T key :)